src/app/onboarding/(components)/onboarding-form.tsx

```
@@ -0,0 +1,167 @@
+              </div>
+              <div className="mt-1 text-xs text-gray-500">
+                {stepTitle[i]}
+              </div>
+            </div>
+          ))}
+        </div>
+        <div className="mt-2 h-1 w-full bg-gray-200">
+          <div
+            className="bg-primary h-1 transition-all duration-300 ease-in-out"
+            style={{ width: `${((step - 1) / 2) * 100}%` }}
+          ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)